__tests__/app.test.js

```diff
@@ -82,13 +83,31 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
-describe("GET any incorrect endpoint", () => {
-  test("when passed an incorrect endpoint, returns 404 error with error message", () => {
+describe("GET /api/articles", () => {
+  test("200: sends an array of correctly formatted articles to the client in descending order based on their creation date", () => {
     return request(app)
-      .get("/topics")
-      .expect(404)
+      .get("/api/articles")
+      .expect(200)
       .then((response) => {
-        expect(response.body.message).toBe("Path not found.");
+        const articlesArray = response.body.articles;
+        console.log(articlesArray);
+        expect(articlesArray.length).toBe(13);
+
+        expect(articlesArray).toBeSortedBy("created_at", {
+          descending: true,
+          coerce: true,
+        });
+        articlesArray.forEach((article) => {
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
```

Comment:
You should look to use the jest matchers, `toHaveProperty` in tandem with `expect.any()`